db/migrate/20240122104018_add_eia_information_to_planning_application.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddEiaInformationToPlanningApplication < ActiveRecord::Migration[7.0]
+  def change
+    add_column :planning_applications, :eia_address, :string
+    add_column :planning_applications, :eia_fee, :integer
+  end
```

Comment:
Might we want to create an EIA table at this rate? Only applies to certain application types too so might be worthwhile

Think there's 3 columns now associated with it